documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -795,13 +801,59 @@ EOF
 <7> Specify the name of the `StorageMap` CR.
-<8> By default, virtual network interface controllers (vNICs) change during the migration process. As a result, vNICs that are configured with a static IP linked to the interface name in the guest VM lose their IP.
+<8> Generates a network interface name for a target VM that overrides the template set in the plan.
+The template follows the Go template syntax and has access to the following variables:
+* `.NetworkName:` If the target network is `multus`, add the name of the Multus Network Attachment Definition. Otherwise, leave this variable empty.
+* `.NetworkNamespace`: If the target network is `multus`, add the namespace where the Multus Network Attachment Definition is located.
+* `.NetworkType`: Specifies the network type. Options: `multus` or `pod`.
+* `.NetworkIndex`: Sequential index of the network interface (0-based).
+Examples
+* `"net-{{.NetworkIndex}}"`
+* `{{if eq .NetworkType "pod"}}pod{{else}}multus-{{.NetworkIndex}}{{end}}"`
++
+Variable names cannot exceed 63 characters. This rules to a network name network templat, a PVC name template, a VM name template, and to a volume name template. 
+<9> By default, virtual network interface controllers (vNICs) change during the migration process. As a result, vNICs that are configured with a static IP address linked to the interface name in the guest VM lose their IP address.
 To avoid this, set `preserveStaticIPs` to `true`. {project-short} issues a warning message about any VMs for which vNIC properties are missing. To retrieve any missing vNIC properties, run those VMs in vSphere in order for the vNIC properties to be reported to {project-short}.
-<9> You can use either the `id` or the `name` parameter to specify the source VMs.
-<10> Specify the VMware vSphere VM moRef. To retrieve the moRef, see xref:retrieving-vmware-moref_vmware[Retrieving a VMware vSphere moRef].
-<11> Optional: You can specify up to two hooks for a VM. Each hook must run during a separate migration step.
-<12> Specify the name of the `Hook` CR.
-<13> Allowed values are `PreHook`, before the migration plan starts, or `PostHook`, after the migration is complete.
+<10> Generates a persistent volume claim (PVC) name for a VM disk that overrides the template set in the plan.
+The template follows the Go template syntax and has access to the following variables:
+* `.VnName`: Name of the VM.
+* `.PlanName`: Name of the migration plan.
+* `.DiskIndex`: Initial volume index of the disk.
+* `.RootDiskIndex`: Index of the root disk.
+Examples
+* `"{{.VmName}}-disk-{{.DiskIndex}}"`
+* `"{{if eq .DiskIndex .RootDiskIndex}}root{{else}}data{{end}}-{{.DiskIndex}}"`
++
+Variable names cannot exceed 63 characters.
+<11> Specify the VMware vSphere VM moRef. To retrieve the moRef, see xref:retrieving-vmware-moref_vmware[Retrieving a VMware vSphere moRef].
+<12> Optional.
+<13> As stated in callout 8: +
```

Comment:
flowing is a copy of 8 , so we don't really need to say as stated in 8, ( maybe we can say "as in 8" and not copy the text here again ? )